dev/Dockerfile

```diff
@@ -0,0 +1,22 @@
+from openms/executables:latest
```

Comment:
I know that this is a dev profile, but 'latest' might break things silently.
Are development versions of OpenMS versionized? Like SNAPSHOT versions or something? If yes I would suggest to use them.

Reply:
Unfortunately, not yet. We are working on nightly conda builds and I will try to add a date as version.